developers/advanced/ingestion-level-aggregations.md

```diff
@@ -136,13 +136,14 @@ From the below aggregation config example, note that `price`  exists in the inpu
 
 ## Allowed Aggregation Functions
 
-| function name    | notes                              |
-| ---------------- | ---------------------------------- |
-| MAX              |                                    |
-| MIN              |                                    |
-| SUM              |                                    |
-| COUNT            | Specify as `COUNT(*)`              |
-| DISTINCTCOUNTHLL | Not available yet, but coming soon |
+| function name    | notes                                                      |
+| ---------------- | ---------------------------------------------------------- |
+| MAX              |                                                            |
+| MIN              |                                                            |
+| SUM              |                                                            |
+| COUNT            | Specify as `COUNT(*)`                                      |
+| DISTINCTCOUNTHLL | Specify as `DISTINCTCOUNTHLL(field, log2m)`, default is 12. Cannot be changed later, a new field must be used. The shcema for the output field should be `BYTES` type. | 
```

Comment:
We also support DISTINCTCOUNTHLLPLUS, consider adding it as well?

Reply:
updated